modules/webbrowser/tests/regression/web-property-sync/script.py

```diff
@@ -9,7 +9,7 @@
 while (lineWidth.value != 2): 
     inviwopy.qt.update();
 # HACK: Wait for as little as possible while ensuring that the webpage has re-rendered. 
-time.sleep(0.1);
+time.sleep(1.5);
```

Comment:
Don’t forget to update this script as well :)

Reply:
I thought the purpose of this test is to check whether the property actually changes. So I would not change this one to wait for `isLoading`